### ISSUE TITLE
Applying twind style on dynamic html

### ISSUE THREAD

**Issue by Johnatan85** [denoland/fresh#1775]
I'm trying to apply Twind styling to HTML content that is dynamically inserted into my fresh app. I'm using the dangerouslySetInnerHTML prop to insert the HTML content, but I'm having trouble applying Twind classes to the elements in the HTML content.

I tried using the `inline` method but that didn't work.

```tsx
import { PageProps } from "$fresh/server.ts";

export default function TestPage(props: PageProps) {
	const dynamicHtml = "<p class='text-red-500'>Hello</p>";
	return (
		<main>
			<div dangerouslySetInnerHTML={{
				__html: dynamicHtml
			}}></div>

		</main>
	);
}
```

Im using twindv1

**Comment by marvinhagemeister** [comment]
Calls to `.innerHTML` are invisible for twind. To work around that, twind has a `tw` helper.

```jsx
const dynamicHtml = `<p class="${tw`text-red-500`}">Hello</p>`;
```

**Comment by Johnatan85** [comment]
That means I would have to parse the html extract the classes and apply the `tw` helper to all of them? 
Is there any way other way do it automatically?

**Comment by Johnatan85** [comment]
Found this https://twind.style/packages/@twind/core#extract 
I guess I could use it for my use case to extract the CSS but is there a way to update the fresh generated style tag manually? 

```html
<style id="__FRSH_TWIND"></style>
```

**Comment by marvinhagemeister** [comment]
Closing this as we've switched to Tailwind CSS which allows us to compile the style sheet ahead of time and move it out of the render path. The Tailwind CSS plugin will be released in the next Fresh 1.6 version which I'm hoping to release later this week. We're likely going to keep the twind plugin on life support and deprecate it soon.
